tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1035,3 +1035,97 @@ Feature: check share activity
         }
       }
       """
+
+  @issue-9856
+  Scenario: check activitiy message after language translation
```

Comment:
Also, please move the scenario to another activities file.

Reply:
@saw-jan Sure, any suggestions for the another activities file's name?

Reply:
May be just move `apiGraph/activities.feature` to `apiActivities/activities.feature` and add the scenarios there